polyglot_piranha.pyi

```diff
@@ -51,26 +51,34 @@ class PiranhaArguments:
 
     def __init__(
         self,
-        path_to_codebase: str,
-        path_to_configuration: str,
         language: str,
         substitutions: dict,
-        **kwargs
+        path_to_configurations: Optional[str],
+        rule_graph: Optional[RuleGraph]= None,
+        path_to_codebase: Optional[str] = None,
+        code_snippet: Optional[str] = None,
+        dry_run: Optional[bool] = None,
+        cleanup_comments: Optional[bool] = None,
+        cleanup_comments_buffer: Optional[int] = None,
+        number_of_ancestors_in_parent_scope: Optional[int] = None,
+        delete_file_if_empty : Optional[bool] = None,
+        delete_consecutive_new_lines : Optional[bool] = None,
+        path_to_output: Optional[str] = None
+ 
     ):
         """
         Constructs `PiranhaArguments`
 
         Parameters
         ------------
-            path_to_codebase: str
-                Path to source code folder or file
-            path_to_configurations: str
-                 Directory containing the configuration files - `piranha_arguments.toml`, `rules.toml`, and  `edges.toml` (optional)
             language: str
                 the target language
             substitutions: dict
                  Substitutions to instantiate the initial set of rules
             keyword arguments: _
+                 path_to_codebase (str) : Path to source code folder or file
+                 path_to_configurations (str) : Directory containing the configuration files - `piranha_arguments.toml`, `rules.toml`, and  `edges.toml`
+                 rule_graph (str) : The rule graph constructed via RuleGraph DSL
```

Comment:
Can we have these be in the same order as in the signature?

Reply:
look at https://github.com/uber/piranha/pull/372/commits/540e83dee67a6d6f653817d8ae3240ceca6a61c4